src/Adyen/Service/AbstractResource.php

```diff
@@ -111,7 +122,7 @@ private function addDefaultParametersToRequest($params)
 	private function handleApplicationInfoInRequest($params)
 	{
 		// Only add if allowed
-		if ($this->allowApplicationInfo) {
+		if ($this->allowApplicationInfo || $this->allowApplicationInfoPOS) {
```

Comment:
I believe that this will always be true.

If I understood the code correctly, this method will only be reached by lines 71 to 75, which are doing exactly this validation.